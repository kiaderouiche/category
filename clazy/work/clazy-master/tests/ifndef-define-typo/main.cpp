
#ifndef GL_FRAMEBUFFER_SRB
# define GL_FRAMEBUFFER_SRGB // Warn
#endif

#ifndef GL_FRAMEBUFFER_SRB
# define GL_FRAMEBUFFER_SRB // OK
# define GL_FRAMEBUFFER_SRGB // OK
#endif

#ifndef AA
# define AAA // OK, too small
#endif


#ifndef GL_FRAMEBUFFER_SRGB
# define GL_FRAMEBUFFER_SRGB // OK
#endif
