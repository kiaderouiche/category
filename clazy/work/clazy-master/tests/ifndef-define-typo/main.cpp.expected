ifndef-define-typo/main.cpp:3:10: warning: Possible typo in define. GL_FRAMEBUFFER_SRB vs GL_FRAMEBUFFER_SRGB [-Wclazy-ifndef-define-typo]
